i18n/states.php

```diff
@@ -1735,6 +1735,33 @@
 		'AE' => __( 'Armed Forces (AE)', 'woocommerce' ),
 		'AP' => __( 'Armed Forces (AP)', 'woocommerce' ),
 	),
+	'VE' => array( // Venezuela States. Ref: https://en.wikipedia.org/wiki/ISO_3166-2:VE
```

Comment:
Please note that WooCommerce use CLDR instead of ISO, so please update all states to reflect this page: https://github.com/unicode-org/cldr/blob/release-38-1/common/subdivisions/en.xml#L5426-L5451
You can keep just one letter for the code, but need to update for the English names, so it's easy to just translate later.

Reply:
@claudiosanches Updated the standard used, any idea why the 'Run code sniff on PR / Code sniff' is failing on a line I didn't add?

Looks to be:
`Line 1299 'SV' => __( 'Suceava', 'woocommerce' ),`

Reply:
@jjchrisdiehl that error is fine for now. Thanks.